package.json

```
@@ -6,7 +6,7 @@
     "bankid",
     "authentication"
   ],
-  "version": "3.2.1",
+  "version": "4.0.0",
```

I think a major bump would incorrectly signaling breaking changes, everything here is backward compatible so minor would suffice. 

```suggestion
  "version": "3.3.0",
```